tests/test_adapter.py

```diff
@@ -88,7 +102,11 @@ def test_aside(self, mocked_allure, narrator_level, allure_level):
         mocked_allure.step.assert_called_once_with(aside_message)
         mocked_allure.severity.assert_called_once_with(allure_level)
 
-    def test_error(self, mocked_manager, mock_allure_trappings):
+    def test_error(
+        self,
+        mocked_manager: mock.Mock,  # noqa: ARG002
```

Comment:
I think this `mocked_manager` was actually an accidental copy-paste error, or maybe i forgot to delete it after changing strategies. You can (probably) remove it!

I don't know why `pytest` hasn't been complaining about an unknown fixture though.

Reply:
I didn't dig into it, but removing it causes the test to fail and I don't know why.  I figured there was some magic going on.

Reply:
ahh. found it.  It shouldn't be called mocked_manager. It's technically the mocked_allure.  The patch is happening at the class level. I updated it to make more sense.